consensus/polybft/consensus_runtime.go

```diff
@@ -856,6 +856,29 @@ func (c *consensusRuntime) GenerateExitProof(exitID, epoch, checkpointBlock uint
 	return tree.GenerateProofForLeaf(e, 0)
 }
 
+// GetStateSyncProof returns the proof of the bundle for the state sync
+func (c *consensusRuntime) GetStateSyncProof(stateSyncID uint64) (*types.StateSyncProof, error) {
+	bundlesToExecute, err := c.state.getBundles(stateSyncID, 1)
+	if err != nil {
+		return nil, fmt.Errorf("cannot get bundles: %w", err)
+	}
+
+	if len(bundlesToExecute) == 0 {
+		return nil, fmt.Errorf("cannot find bundle containing StateSync id %d", stateSyncID)
+	}
+
+	for _, bundle := range bundlesToExecute[0].StateSyncs {
+		if bundle.ID == stateSyncID {
```

Comment:
Small thing, I think we don't need this condition, since we have only one state sync event per bundle now, so it will contain only the one we are looking for.

Reply:
Right, this was leftover from previous tests, but let's leave it as is at this time just in case, we'll remove it together with bundles in next PRs.